README.md

```diff
@@ -33,6 +33,8 @@ ambta_doctrine_encrypt:
 
 ### Secret key
 
+The secret key should be a max 32 byte hexadecimal string (`[0-9a-fA-F]`).
```

Comment:
why "should"? if it does not work with other strings, i'd say "The secret key needs to be a hexadecimal string (...) of maximum 32 bytes."

Reply:
Agreed :)